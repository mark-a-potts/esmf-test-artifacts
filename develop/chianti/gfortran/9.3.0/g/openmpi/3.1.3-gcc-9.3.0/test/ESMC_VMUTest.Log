build time -- 2021-04-23 12:43:04.760193
20210423 124705.711 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.711 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.711 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.711 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.711 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.711 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.711 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.711 INFO             PET0 ESMF_COMM                   : openmpi
20210423 124705.711 INFO             PET0 ESMF_MOAB                   : enabled
20210423 124705.711 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 124705.711 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 124705.711 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 124705.711 INFO             PET0 ESMF_PIO                    : enabled
20210423 124705.711 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 124705.711 INFO             PET0 Beginning Test, file ESMC_VMUTest.C, line 44
20210423 124705.711 INFO             PET0 NUMBER_OF_PROCESSORS 4
20210423 124705.711 INFO             PET0 PASS VMGetGlobal, ESMC_VMUTest.C, line 59
20210423 124705.711 INFO             PET0 PASS VMGetCurrent, ESMC_VMUTest.C, line 68
20210423 124705.711 INFO             PET0 PASS VMGet, ESMC_VMUTest.C, line 78
20210423 124705.711 INFO             PET0 PASS VMBarrier, ESMC_VMUTest.C, line 90
20210423 124705.711 INFO             PET0 PASS VMPrint, ESMC_VMUTest.C, line 99
20210423 124705.711 INFO             PET0 PASS VMReduceI4, ESMC_VMUTest.C, line 124
20210423 124705.711 INFO             PET0 PASS VMReduceI8, ESMC_VMUTest.C, line 150
20210423 124705.712 INFO             PET0 PASS VMReduceR4, ESMC_VMUTest.C, line 176
20210423 124705.712 INFO             PET0 PASS VMReduceR8, ESMC_VMUTest.C, line 202
20210423 124705.712 INFO             PET0 PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
20210423 124705.712 INFO             PET0 PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
20210423 124705.712 INFO             PET0 PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
20210423 124705.712 INFO             PET0 PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
20210423 124705.712 INFO             PET0 PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
20210423 124705.712 INFO             PET0 Ending Test, file ESMC_VMUTest.C, line 358
20210423 124705.712 INFO             PET0 Finalizing ESMF
20210423 124705.711 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.711 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.711 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.711 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.711 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.711 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.711 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.711 INFO             PET1 ESMF_COMM                   : openmpi
20210423 124705.711 INFO             PET1 ESMF_MOAB                   : enabled
20210423 124705.711 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 124705.711 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 124705.711 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 124705.711 INFO             PET1 ESMF_PIO                    : enabled
20210423 124705.711 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 124705.711 INFO             PET1 Beginning Test, file ESMC_VMUTest.C, line 44
20210423 124705.711 INFO             PET1 NUMBER_OF_PROCESSORS 4
20210423 124705.711 INFO             PET1 PASS VMGetGlobal, ESMC_VMUTest.C, line 59
20210423 124705.711 INFO             PET1 PASS VMGetCurrent, ESMC_VMUTest.C, line 68
20210423 124705.711 INFO             PET1 PASS VMGet, ESMC_VMUTest.C, line 78
20210423 124705.711 INFO             PET1 PASS VMBarrier, ESMC_VMUTest.C, line 90
20210423 124705.711 INFO             PET1 PASS VMPrint, ESMC_VMUTest.C, line 99
20210423 124705.711 INFO             PET1 PASS VMReduceI4, ESMC_VMUTest.C, line 124
20210423 124705.711 INFO             PET1 PASS VMReduceI8, ESMC_VMUTest.C, line 150
20210423 124705.711 INFO             PET1 PASS VMReduceR4, ESMC_VMUTest.C, line 176
20210423 124705.711 INFO             PET1 PASS VMReduceR8, ESMC_VMUTest.C, line 202
20210423 124705.712 INFO             PET1 PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
20210423 124705.712 INFO             PET1 PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
20210423 124705.712 INFO             PET1 PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
20210423 124705.712 INFO             PET1 PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
20210423 124705.712 INFO             PET1 PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
20210423 124705.712 INFO             PET1 Ending Test, file ESMC_VMUTest.C, line 358
20210423 124705.712 INFO             PET1 Finalizing ESMF
20210423 124705.711 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.711 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.711 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.711 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.711 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.711 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.711 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.711 INFO             PET2 ESMF_COMM                   : openmpi
20210423 124705.711 INFO             PET2 ESMF_MOAB                   : enabled
20210423 124705.711 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 124705.711 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 124705.711 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 124705.711 INFO             PET2 ESMF_PIO                    : enabled
20210423 124705.711 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 124705.711 INFO             PET2 Beginning Test, file ESMC_VMUTest.C, line 44
20210423 124705.711 INFO             PET2 NUMBER_OF_PROCESSORS 4
20210423 124705.711 INFO             PET2 PASS VMGetGlobal, ESMC_VMUTest.C, line 59
20210423 124705.711 INFO             PET2 PASS VMGetCurrent, ESMC_VMUTest.C, line 68
20210423 124705.711 INFO             PET2 PASS VMGet, ESMC_VMUTest.C, line 78
20210423 124705.711 INFO             PET2 PASS VMBarrier, ESMC_VMUTest.C, line 90
20210423 124705.711 INFO             PET2 PASS VMPrint, ESMC_VMUTest.C, line 99
20210423 124705.711 INFO             PET2 PASS VMReduceI4, ESMC_VMUTest.C, line 124
20210423 124705.711 INFO             PET2 PASS VMReduceI8, ESMC_VMUTest.C, line 150
20210423 124705.711 INFO             PET2 PASS VMReduceR4, ESMC_VMUTest.C, line 176
20210423 124705.712 INFO             PET2 PASS VMReduceR8, ESMC_VMUTest.C, line 202
20210423 124705.712 INFO             PET2 PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
20210423 124705.712 INFO             PET2 PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
20210423 124705.712 INFO             PET2 PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
20210423 124705.712 INFO             PET2 PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
20210423 124705.712 INFO             PET2 PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
20210423 124705.712 INFO             PET2 Ending Test, file ESMC_VMUTest.C, line 358
20210423 124705.712 INFO             PET2 Finalizing ESMF
20210423 124705.711 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.711 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.711 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.711 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.711 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.711 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.711 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.711 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.711 INFO             PET3 ESMF_COMM                   : openmpi
20210423 124705.711 INFO             PET3 ESMF_MOAB                   : enabled
20210423 124705.711 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 124705.711 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 124705.711 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 124705.711 INFO             PET3 ESMF_PIO                    : enabled
20210423 124705.711 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 124705.711 INFO             PET3 Beginning Test, file ESMC_VMUTest.C, line 44
20210423 124705.711 INFO             PET3 NUMBER_OF_PROCESSORS 4
20210423 124705.711 INFO             PET3 PASS VMGetGlobal, ESMC_VMUTest.C, line 59
20210423 124705.711 INFO             PET3 PASS VMGetCurrent, ESMC_VMUTest.C, line 68
20210423 124705.711 INFO             PET3 PASS VMGet, ESMC_VMUTest.C, line 78
20210423 124705.711 INFO             PET3 PASS VMBarrier, ESMC_VMUTest.C, line 90
20210423 124705.711 INFO             PET3 PASS VMPrint, ESMC_VMUTest.C, line 99
20210423 124705.711 INFO             PET3 PASS VMReduceI4, ESMC_VMUTest.C, line 124
20210423 124705.711 INFO             PET3 PASS VMReduceI8, ESMC_VMUTest.C, line 150
20210423 124705.711 INFO             PET3 PASS VMReduceR4, ESMC_VMUTest.C, line 176
20210423 124705.711 INFO             PET3 PASS VMReduceR8, ESMC_VMUTest.C, line 202
20210423 124705.712 INFO             PET3 PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
20210423 124705.712 INFO             PET3 PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
20210423 124705.712 INFO             PET3 PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
20210423 124705.712 INFO             PET3 PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
20210423 124705.712 INFO             PET3 PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
20210423 124705.712 INFO             PET3 Ending Test, file ESMC_VMUTest.C, line 358
20210423 124705.712 INFO             PET3 Finalizing ESMF
