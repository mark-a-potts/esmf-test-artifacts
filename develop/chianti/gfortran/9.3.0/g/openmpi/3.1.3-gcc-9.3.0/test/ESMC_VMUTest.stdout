build time -- 2021-04-23 12:43:04.760193
localPet= 0 , petCount= 4, peCount= 4, pthreadsEnabledFlag= 1, openMPEnabledFlag= 1 
--- ESMCI::VM::print() start ---
--- ESMCI::VMId::print() start ---
  vmKeyWidth = 1
  vmKey=0xF0000000
  localID = 0
--- ESMCI::VMId::print() end ---
--- VMK::print() start ---
vm located at: 0x14d88a0
npets = 4, mypet=0
  pth_mutex =		 0x1602a10
  pth_finish_count =	 (nil)
MPI_Comm_size: 4
MPI_Comm_rank in local MPI communicator: 0
MPI_Comm_rank in MPI_COMM_WORLD: 0
MPI thread level support: 3
mpi_mutex_flag: 0
mpionly: 1
threadsflag: 0
  lpid[0]=0, tid[0]=0, vas[0]=0, ncpet[0]=1, nadevs[0]=0, cid[0][0]=0
  lpid[1]=1, tid[1]=0, vas[1]=1, ncpet[1]=1, nadevs[1]=0, cid[1][0]=1
  lpid[2]=2, tid[2]=0, vas[2]=2, ncpet[2]=1, nadevs[2]=0, cid[2][0]=2
  lpid[3]=3, tid[3]=0, vas[3]=3, ncpet[3]=1, nadevs[3]=0, cid[3][0]=3
ncores = 4
  cpuid[0]=0, ssiid[0]=0
  cpuid[1]=1, ssiid[1]=0
  cpuid[2]=2, ssiid[2]=0
  cpuid[3]=3, ssiid[3]=0
--- VMK::print() end ---
--- ESMCI::VM::print() end ---
localPet= 1 , petCount= 4, peCount= 4, pthreadsEnabledFlag= 1, openMPEnabledFlag= 1 
--- ESMCI::VM::print() start ---
--- ESMCI::VMId::print() start ---
  vmKeyWidth = 1
  vmKey=0xF0000000
  localID = 0
--- ESMCI::VMId::print() end ---
--- VMK::print() start ---
vm located at: 0xb748a0
npets = 4, mypet=1
  pth_mutex =		 0xc9aee0
  pth_finish_count =	 (nil)
MPI_Comm_size: 4
MPI_Comm_rank in local MPI communicator: 1
MPI_Comm_rank in MPI_COMM_WORLD: 1
MPI thread level support: 3
mpi_mutex_flag: 0
mpionly: 1
threadsflag: 0
  lpid[0]=0, tid[0]=0, vas[0]=0, ncpet[0]=1, nadevs[0]=0, cid[0][0]=0
  lpid[1]=1, tid[1]=0, vas[1]=1, ncpet[1]=1, nadevs[1]=0, cid[1][0]=1
  lpid[2]=2, tid[2]=0, vas[2]=2, ncpet[2]=1, nadevs[2]=0, cid[2][0]=2
  lpid[3]=3, tid[3]=0, vas[3]=3, ncpet[3]=1, nadevs[3]=0, cid[3][0]=3
ncores = 4
  cpuid[0]=0, ssiid[0]=0
  cpuid[1]=1, ssiid[1]=0
  cpuid[2]=2, ssiid[2]=0
  cpuid[3]=3, ssiid[3]=0
--- VMK::print() end ---
--- ESMCI::VM::print() end ---
Beginning Test, file ESMC_VMUTest.C, line 44
NUMBER_OF_PROCESSORS 4
PASS VMGetGlobal, ESMC_VMUTest.C, line 59
PASS VMGetCurrent, ESMC_VMUTest.C, line 68
PASS VMGet, ESMC_VMUTest.C, line 78
PASS VMBarrier, ESMC_VMUTest.C, line 90
PASS VMPrint, ESMC_VMUTest.C, line 99
PASS VMReduceI4, ESMC_VMUTest.C, line 124
PASS VMReduceI8, ESMC_VMUTest.C, line 150
PASS VMReduceR4, ESMC_VMUTest.C, line 176
PASS VMReduceR8, ESMC_VMUTest.C, line 202
PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
Ending Test, file ESMC_VMUTest.C, line 358
Beginning Test, file ESMC_VMUTest.C, line 44
NUMBER_OF_PROCESSORS 4
PASS VMGetGlobal, ESMC_VMUTest.C, line 59
PASS VMGetCurrent, ESMC_VMUTest.C, line 68
PASS VMGet, ESMC_VMUTest.C, line 78
PASS VMBarrier, ESMC_VMUTest.C, line 90
PASS VMPrint, ESMC_VMUTest.C, line 99
PASS VMReduceI4, ESMC_VMUTest.C, line 124
PASS VMReduceI8, ESMC_VMUTest.C, line 150
PASS VMReduceR4, ESMC_VMUTest.C, line 176
PASS VMReduceR8, ESMC_VMUTest.C, line 202
PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
Ending Test, file ESMC_VMUTest.C, line 358
localPet= 2 , petCount= 4, peCount= 4, pthreadsEnabledFlag= 1, openMPEnabledFlag= 1 
--- ESMCI::VM::print() start ---
--- ESMCI::VMId::print() start ---
  vmKeyWidth = 1
  vmKey=0xF0000000
  localID = 0
--- ESMCI::VMId::print() end ---
--- VMK::print() start ---
vm located at: 0x228f8a0
npets = 4, mypet=2
  pth_mutex =		 0x23b98d0
  pth_finish_count =	 (nil)
MPI_Comm_size: 4
MPI_Comm_rank in local MPI communicator: 2
MPI_Comm_rank in MPI_COMM_WORLD: 2
MPI thread level support: 3
mpi_mutex_flag: 0
mpionly: 1
threadsflag: 0
  lpid[0]=0, tid[0]=0, vas[0]=0, ncpet[0]=1, nadevs[0]=0, cid[0][0]=0
  lpid[1]=1, tid[1]=0, vas[1]=1, ncpet[1]=1, nadevs[1]=0, cid[1][0]=1
  lpid[2]=2, tid[2]=0, vas[2]=2, ncpet[2]=1, nadevs[2]=0, cid[2][0]=2
  lpid[3]=3, tid[3]=0, vas[3]=3, ncpet[3]=1, nadevs[3]=0, cid[3][0]=3
ncores = 4
  cpuid[0]=0, ssiid[0]=0
  cpuid[1]=1, ssiid[1]=0
  cpuid[2]=2, ssiid[2]=0
  cpuid[3]=3, ssiid[3]=0
--- VMK::print() end ---
--- ESMCI::VM::print() end ---
localPet= 3 , petCount= 4, peCount= 4, pthreadsEnabledFlag= 1, openMPEnabledFlag= 1 
--- ESMCI::VM::print() start ---
--- ESMCI::VMId::print() start ---
  vmKeyWidth = 1
  vmKey=0xF0000000
  localID = 0
--- ESMCI::VMId::print() end ---
--- VMK::print() start ---
vm located at: 0x171c8a0
npets = 4, mypet=3
  pth_mutex =		 0x18467a0
  pth_finish_count =	 (nil)
MPI_Comm_size: 4
MPI_Comm_rank in local MPI communicator: 3
MPI_Comm_rank in MPI_COMM_WORLD: 3
MPI thread level support: 3
mpi_mutex_flag: 0
mpionly: 1
threadsflag: 0
  lpid[0]=0, tid[0]=0, vas[0]=0, ncpet[0]=1, nadevs[0]=0, cid[0][0]=0
  lpid[1]=1, tid[1]=0, vas[1]=1, ncpet[1]=1, nadevs[1]=0, cid[1][0]=1
  lpid[2]=2, tid[2]=0, vas[2]=2, ncpet[2]=1, nadevs[2]=0, cid[2][0]=2
  lpid[3]=3, tid[3]=0, vas[3]=3, ncpet[3]=1, nadevs[3]=0, cid[3][0]=3
ncores = 4
  cpuid[0]=0, ssiid[0]=0
  cpuid[1]=1, ssiid[1]=0
  cpuid[2]=2, ssiid[2]=0
  cpuid[3]=3, ssiid[3]=0
--- VMK::print() end ---
--- ESMCI::VM::print() end ---
Beginning Test, file ESMC_VMUTest.C, line 44
NUMBER_OF_PROCESSORS 4
PASS VMGetGlobal, ESMC_VMUTest.C, line 59
PASS VMGetCurrent, ESMC_VMUTest.C, line 68
PASS VMGet, ESMC_VMUTest.C, line 78
PASS VMBarrier, ESMC_VMUTest.C, line 90
PASS VMPrint, ESMC_VMUTest.C, line 99
PASS VMReduceI4, ESMC_VMUTest.C, line 124
PASS VMReduceI8, ESMC_VMUTest.C, line 150
PASS VMReduceR4, ESMC_VMUTest.C, line 176
PASS VMReduceR8, ESMC_VMUTest.C, line 202
PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
Ending Test, file ESMC_VMUTest.C, line 358
Beginning Test, file ESMC_VMUTest.C, line 44
NUMBER_OF_PROCESSORS 4
PASS VMGetGlobal, ESMC_VMUTest.C, line 59
PASS VMGetCurrent, ESMC_VMUTest.C, line 68
PASS VMGet, ESMC_VMUTest.C, line 78
PASS VMBarrier, ESMC_VMUTest.C, line 90
PASS VMPrint, ESMC_VMUTest.C, line 99
PASS VMReduceI4, ESMC_VMUTest.C, line 124
PASS VMReduceI8, ESMC_VMUTest.C, line 150
PASS VMReduceR4, ESMC_VMUTest.C, line 176
PASS VMReduceR8, ESMC_VMUTest.C, line 202
PASS VMBroadcastI4, ESMC_VMUTest.C, line 227
PASS VMBroadcastI8, ESMC_VMUTest.C, line 252
PASS VMBroadcastR4, ESMC_VMUTest.C, line 277
PASS VMBroadcastR8, ESMC_VMUTest.C, line 302
PASS VMBroadcastLogical, ESMC_VMUTest.C, line 327
Ending Test, file ESMC_VMUTest.C, line 358
 PET 0 Test Elapsed Time  1.575000 msec.
 PET 2 Test Elapsed Time  1.579000 msec.
 PET 3 Test Elapsed Time  1.572000 msec.
 PET 1 Test Elapsed Time  3.637000 msec.
